modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,425 @@
+= Configure GCP Private Service Connect for BYOC
+:description: Set up GCP Private Service Connect to securely access Redpanda Cloud. 
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+// This guide describes the steps for setting up Private Service Connect (PSC) on Redpanda Cloud.
+
+// Redpanda GCP PSC can be enabled during cluster creation or for an existing cluster.
+
+// After completing this guide, a PSC service attachment will have been deployed for your Redpanda Cloud cluster. You may then establish connections to it from any GCP project included in the allow list.
+
+After <<get-a-cloud-api-access-token,getting an access token>>, you can <<create-new-byovpc-cluster-with-psc-enabled,enable PSC when creating a new BYOC cluster>>, or you can <<enable-psc-on-an-existing-byovpc-cluster,enable PSC for existing BYOC clusters>>.
+
+== Requirements
+
+* You must use the Redpanda Cloud API to enable the Redpanda endpoint service for your clusters. Follow the steps below to <<get-a-cloud-api-access-token,get an access token>>. 
```

Comment:
Also want to double check whether the requirements for using PSC might be similar to what we listed for PL: https://docs.redpanda.com/current/deploy/deployment-option/cloud/aws-privatelink/#requirements